migrations/913-websites-lookup-perms.sql

```diff
@@ -0,0 +1 @@
+UPDATE groups SET rules=CONCAT(rules, ',WebsiteLookup:View') WHERE name IN ('Staff', 'Support Staff', 'Carriers and Operators');
```

Comment:
:+1: 
we probably should have started out doing `Lookup:App` , etc in the beginning imo.